### PR TITLE
Migrate stories to TS

### PR DESCRIPTION
**Pull request by mickaelzhang**
## What does this do?

- Moved all stories and file to TS
- Only js file are now config one

### CODE REVIEW

File: src/Accordion/Accordion.stories.tsx [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-import { Accordion } from './Accordion'
+import { Accordion, AccordionProps } from './Accordion'
 
 export default {
   title: 'Accordion',
   component: Accordion,
 }
 
-const Template = (args) => (
+const Template = (args: AccordionProps) => (
```
Comment (Jmuskett) [inline src/Accordion/Accordion.stories.tsx:9]: super nit, but shouldn't this be

```
props: AccordionProps
```

OR

```
args: accordionArgs
```
?

File: src/Box/Box.stories.tsx [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-import { Box } from './Box'
+import { Box, BoxProps } from './Box'
 
 export default {
   title: 'Box',
   component: Box,
 }
 
-const Template = (args) => <Box {...args}>Some lovely content.</Box>
+const Template = (args: BoxProps) => <Box {...args}>Some lovely content.</Box>
```
Comment (Jmuskett) [inline src/Box/Box.stories.tsx:9]: same comment as above, I think let's go for consistent naming on these?

**Review by Jmuskett — CHANGES_REQUESTED** [review]
looks good on the whole, just a question about where we are using `args` but typing it as `props` - I know it's a real nit but I think we should be consistent with the naming, either props or args

File: src/Accordion/Accordion.stories.tsx (hunk 1 above)
Comment (mickaelzhang) [reply src/Accordion/Accordion.stories.tsx:9]: I didn't think about changing anything in the stories, except when it was related to typing
The typing is related to props so it should definitely be `AccordionProps`
For the other one, I don't mind either way, I'll update it to `props` 👍

**Pull request merged**